docs/admin/how_to_release.md

```diff
@@ -1,96 +1,39 @@
----
-layout: longpage
-title: How to Release
-permalink: /dev_docs/admin/how_to_release
----
 
+## Manual Release Testing
+
+- Can download maps, and can see and load both maps that come with triplea and downloaded maps.
+- Launch an AI only game and let some AI players go at for a dozen rounds, verify no errors
+- Join a bot, play through a combat
+- Compatibility checks
+  - save games
+  - network play
+  - XML parsing
+  
+- Local host + client
+  - This means start up two copies of triplea.  In one copy you click ‘host networked game’ 
+and start hosting.  In the other copy you click ‘connect to networked game’ and join.  The client should choose the 
+FIRST player in the list (this is to test that the delegates are sending data correctly).  
+Start the game and play a round against yourself, making sure everything looks good.  
+
+- Play by Email, and Play by Forum  
+  - You can test both at once by setting up a game against yourself that has both settings enabled.  You will probably need to add two email addresses for this to work.  I also recommend testing the play by forum against both AxisAndAllies.org and TripleAWarClub.org, and also testing the play by email against both a gmail and a hotmail email account.  
```

Comment:
war club links should pretty much all be removed at this point. PBF should be tested against A&A.org and the new forum.